app/views/user_mailer/new_device_sign_in.html.erb

```diff
@@ -18,13 +18,15 @@
 
 <p>
   <%= t(
-        'user_mailer.new_device_sign_in.info_html',
+        'user_mailer.new_device_sign_in.info',
         app_name: APP_NAME,
-        date: @login_date,
-        location: @login_location,
-      ) %>
+      ) %>    
 </p>
 
+<br/>
+
+<p><strong><%= @login_date %><br/><%= @login_location %>, <%= sanitize @device_name %></strong></p>
```

Comment:
I don't think we want `sanitize` since device name shouldn't have any HTML in it
```suggestion
<p><strong><%= @login_date %><br/><%= @login_location %>, <%= @device_name %></strong></p>
```

Reply:
Removed. The value is user supplied so the line was largely preventative. A library we're using likely matches against known values but I didn't fully verify. Limited chance of a problem.

Reply:
Counter-intuitively, `sanitize` will stop user-provided data from being escaped. It's the same as blindly marking a string as `html_safe`